.jscsrc

```diff
@@ -2,11 +2,20 @@
   "esnext": true,
   "validateQuoteMarks": "'",
   "requireDotNotation": true,
-  "requireCurlyBraces": true,
   "requireCommaBeforeLineBreak": true,
   "requireOperatorBeforeLineBreak": true,
   "disallowTrailingWhitespace": true,
   "disallowMultipleLineStrings": true,
   "disallowMixedSpacesAndTabs": true,
-  "disallowEmptyBlocks": true
-}
+  "disallowEmptyBlocks": true,
+
```

Comment:
IDK, this is a case where yeoman generators are sweet - let me glance over the files